dlhub_sdk/models/servables/python.py

```diff
@@ -163,7 +165,16 @@ def create_model(cls, module, method, autobatch=False, function_kwargs=None):
                 Calls :code:`map(f, list)`
             function_kwargs (dict): Names and values of any other argument of the function to set
                 the values must be JSON serializable.
+            f (object): A function pointer
+        Raises:
+            TypeError: If there is no valid way to process the given arguments
         """
+        if f is not None:
```

Comment:
also, what is supposed to happen if `f` _is_ `None`? Unclear from this logic, please add context

Reply:
this logic (modified in the other branch) describes how to handle the parameters since the information can be gotten from either the pointer or both the `module` and `method`, but not neither.